docs/docs/platform/subscribers.md

```diff
@@ -31,48 +45,96 @@ await novu.subscribers.identify(user.id, {
 
-In some cases, you want to access subscribers to update a specific field or data attribute. For example, when the user changes their email address or personal details.
+A non-exisiting subscriber can be added by sending subscriber data in `to` field of the trigger method. If any subscriber with provided `subscriberId` does not exists, a new subscriber will be created. In this case, subscriber will be created first and then the trigger will be executed synchronously.
+
+<Tabs groupId="language" queryString>
+  <TabItem value="js" label="JavaScript">
 
 ```typescript
 import { Novu } from '@novu/node';
 
 const novu = new Novu(process.env.NOVU_API_KEY);
 
-await novu.subscribers.update(user.id, {
-  email: user.email,
+await novu.trigger('<TEMPLATE_IDENTIFIER>', {
+  to: {
+    subscriberId: '<USER_IDENTIFIER>',
+    email: 'test@email.com',
+    firstName: 'John',
+    lastName: 'Doe',
+    phone: '+9876543210',
+  },
+  payload: {
+    customVariables: 'Test',
+    organization: {
+      logo: 'https://organisation.com/logo.png',
+    },
+  },
 });
 ```
 
-### Updating subscriber credentials
+  </TabItem>
+
+  <TabItem value="php" label="PHP">
+
+  </TabItem>
+</Tabs>
 
-Users who wish to use chat channels will need to set their credentials to be authenticated.
+## Find a subscriber
+
+Any subscriber can be retrieved using a unique subscriber identifier `subscriberId`. Check returned subscriber schema [here](#subscriber-response-schema).
 
 ```typescript
-import { Novu, ChatProviderIdEnum } from '@novu/node';
+import { Novu } from '@novu/node';
 
 const novu = new Novu(process.env.NOVU_API_KEY);
 
-await novu.subscribers.setCredentials('subscriberId', ChatProviderIdEnum.Slack, {
-  webhookUrl: 'webhookUrl',
-});
+await novu.subscribers.update('subscriberId');
 ```
 
-- subscriberId is a custom identifier used when identifying your users within the Novu platform.
-- providerId is a unique provider identifier (we recommend using ChatProviderIdEnum).
-- credentials are the argument you need to be authenticated with your provider workspace. At this point, we support chat messages through webhook, so a webhookUrl is needed to be provided.
+## Update a subscriber
 
-### Removing a subscriber
+In some cases, you want to access a subscriber to update a specific user data field or custom data field. For example, when the users change their email address or personal details.
 
-To remove and stop a subscriber from receiving communication, you call the remove API to delete the subscriber.
+```typescript
+import { Novu } from '@novu/node';
+
+const novu = new Novu(process.env.NOVU_API_KEY);
+
+await novu.subscribers.update(user.id, {
+  // new email
+  email: user.email,
+});
+```
+
+## Delete a subscriber
+
+To stop a subscriber from receiving notifications, you can delete the subscriber. This will hard delete the subscriber means you will not be able to access this subscriber later. You will have to create this subscriber again.
```

Comment:
```suggestion
To stop a subscriber from receiving notifications, you can delete the subscriber. This will hard delete the subscriber and means you will not be able to access this subscriber later. You will have to create this subscriber again.
```
Also we should explain what happens with the existing data (activity feed) when a hard delete happens.

Reply:
@p-fernandez We have deleted property in subscriber response
https://github.com/novuhq/novu/blob/4b0fa1d2e69dd6e44d883d88b5caefc8306d2cb1/apps/api/src/app/subscribers/dtos/subscriber-response.dto.ts#L61

I want to confirm are we hard deleting the subscriber in this API or not?